Dockerfile

```
@@ -12,6 +12,7 @@ RUN --mount=type=cache,id=apk-cache-${TARGETARCH},target=/var/cache/apk \
 	build-base \
 	ca-certificates-bundle \
 	libevent-dev \
+	libnghttp2-dev \
```

```suggestion
	nghttp2-dev \
```
https://pkgs.alpinelinux.org/package/v3.20/main/x86_64/nghttp2-dev